### PR TITLE
Added additional info about blocks managing

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional info about blocks managing
Deleted non-existent button "Save&New"

## Affected documentation pages

https://docs.magento.com/user-guide/cms/block-add.html

whatsnew
Updated the [Adding New Blocks](https://docs.magento.com/user-guide/cms/block-add.html) topic to add new section about removing custom blocks.

### CODE REVIEW

File: src/cms/block-add.md [hunk 1]
```diff
@@ -52,11 +52,28 @@ _Blocks page_
 
 1. Use the [Widget]({% link cms/widget-static-block.md %}) tool to place the completed block on a specific page in your store.
 
+## Delete a block
```
Comment (leslietilling) [inline src/cms/block-add.md:55]:
```suggestion
## Delete a block

```

File: src/cms/block-add.md [hunk 2]
```diff
@@ -52,11 +52,28 @@ _Blocks page_
...
+There are two ways to remove created block. You can remove it from Blocks grid, or from the edit block page.
```
Comment (leslietilling) [inline src/cms/block-add.md:56]:
```suggestion
There are two ways to remove created block. You can remove it from _Blocks_ grid, or from the edit block page.
```

File: src/cms/block-add.md [hunk 3]
```diff
@@ -52,11 +52,28 @@ _Blocks page_
...
+
+## Method 1: Remove a block from the Blocks grid
+
+1. On the _Admin_ sidebar, go to **Content** > _Elements_ > **Blocks**.
+1. Locate the blocks using filters above the grid, select the checkbox of the blocks to be deleted.
```
Comment (leslietilling) [inline src/cms/block-add.md:61]:
```suggestion
1. Locate the blocks using filters above the grid and select the checkbox for one or more blocks to be deleted.
```

File: src/cms/block-add.md [hunk 4]
```diff
@@ -52,11 +52,28 @@ _Blocks page_
...
+1. In the upper-left corner of the list, set **Actions** to `Delete`.
+1. To confirm the action, click <span class="btn">OK</span>.
+
+## Method 2: Remove a block from the edit page
+
+1. On the _Admin_ sidebar, go to **Content** > _Elements_ > **Blocks**.
+1. Find the block to be deleted.
+1. In the **Actions** column choose `Edit`.
```
Comment (leslietilling) [inline src/cms/block-add.md:69]:
```suggestion
1. In the _Actions_ column for the block, click **Select** and choose **Edit**.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just some minor edits for style and clarity

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**